otherlibs/unix/unixLabels.mli

```diff
@@ -1160,7 +1160,7 @@ type sigprocmask_command = Unix.sigprocmask_command =
   | SIG_UNBLOCK
 
 val sigprocmask : mode:sigprocmask_command -> int list -> int list
-(** [sigprocmask ~mode sigs] changes the set of blocked signals.
+(** [sigprocmask mode sigs] changes the set of blocked signals.
```

Comment:
Why did you remove the tilde? It's a labelled argument.

Reply:
Why do you phrase your comments as questions?  It was a cut-and-paste error, obviously.  Fixed.